### PR TITLE
Add auto-enable modules even custom

### PR DESCRIPTION
**Pull request by Vaadasch**
Optimization of previous PR https://github.com/tuxgasy/docker-dolibarr/pull/146 for allowing it to auto-activate custom modules at init.

That was only looking inside /var/www/html/core/modules. Now with this PR, it looks inside custom folders.

The script set country and company name before activating modules, that is better

### CODE REVIEW

File: README.template [hunk 1]
```diff
@@ -139,7 +139,7 @@ It is possible to execute `*.sh`, `*.sql` and/or `*.php` custom file at the end
 ```
 \docker-init.d
 |- custom_script.sql
-|- custom_script.php
+|- custom_script.php  // You should create your own module instead of doing this.
```
Comment (tuxgasy) [inline README.template:142]: I think this comment is not relevant. I used it now to do a quick customization on initialization.

File: docker-init.php [hunk 2]
```diff
@@ -7,43 +7,46 @@
...
+  require_once DOL_DOCUMENT_ROOT.'/core/class/ccountry.class.php';
+  $countryCode = getenv('DOLI_COMPANY_COUNTRYCODE');
+  $country = new Ccountry($db);
+  $res = $country->fetch(0,$countryCode);
+  if ($res > 0 ) {
+      $s = $country->id.':'.$country->code.':'.$country->label;
+      dolibarr_set_const($db, "MAIN_INFO_SOCIETE_COUNTRY", $s, 'chaine', 0, '', $conf->entity);
+      printf('Configuring for country : '.$s."\n");
+      activateModulesRequiredByCountry($country->code);
+      $db->commit();
```
Comment (tuxgasy) [inline docker-init.php:22]: I think we can remove this unnecessary commit.

File: docker-init.php [hunk 3]
```diff
@@ -7,43 +7,46 @@
...
-        activateModulesRequiredByCountry($country->code);
-        $db->commit();
-    }
-    else {
-            printf('Unable to find country '.$countryCode."\n");
-    }
+if (!empty(getenv('DOLI_COMPANY_NAME'))) {
+  $compname = getenv('DOLI_COMPANY_NAME');
+  dolibarr_set_const($db, "MAIN_INFO_SOCIETE_NOM", $compname, 'chaine', 0, '', $conf->entity);
+  $db->commit();
```
Comment (tuxgasy) [inline docker-init.php:32]: Same!

File: docker-run.sh [hunk 4]
```diff
@@ -185,6 +185,7 @@ function initializeDatabase()
         sed -i 's/--.*//g;' ${file}
         mysql -u ${DOLI_DB_USER} -p${DOLI_DB_PASSWORD} -h ${DOLI_DB_HOST} -P ${DOLI_DB_HOST_PORT} ${DOLI_DB_NAME} < ${file} > /dev/null 2>&1
       elif [ "$isExec" == "PHP" ] ; then
+        # You should create your own module and activate it with DOLI_ENABLE_MODULES instead of doing that
```
Comment (tuxgasy) [inline docker-run.sh:188]: Comment not necessary

**Comment by Vaadasch** [comment]
Thanks for your review, @tuxgasy, i'm happy to know that it is useful :smile:

**Review by tuxgasy — APPROVED** [review]
To test.

**Pull request merged**